src/sv-pipeline/java/StitchFragmentedCNVs.java

```diff
@@ -106,18 +100,80 @@ private static boolean genotypesMatch( final VCFParser.Record rec1, final VCFPar
         return true;
     }
 
-    private static OutputStream createOutputStream( final String pathName ) throws IOException {
-        final OutputStream os = pathName.equals("-") ? System.out : new FileOutputStream(pathName);
-        return new BufferedOutputStream(pathName.endsWith(GZ) ? new GZIPOutputStream(os) : os);
+    private static void initCommandLineArgs( final String[] args ) {
+        try {
+            PAD_FACTOR = Double.parseDouble(args[0]);
+        } catch ( final NumberFormatException nfe ) {
+            System.err.println("Can't interpret 1st argument as a floating point number.");
+            System.exit(2);
+        }
+        if ( PAD_FACTOR < .01 || PAD_FACTOR > .99 ) {
```

Comment:
I think this just should be any non-negative number - it's possible to pad exactly 0% (if expecting some events already overlap) or more than 100% if you really wanted to go crazy. `if ( PAD_FACTOR < 0 ) {`